src/EditorFeatures/DiagnosticsTestUtilities/CodeActions/AbstractCodeActionOrUserDiagnosticTest.cs

```diff
@@ -726,56 +685,94 @@ protected static ImmutableArray<CodeAction> FlattenActions(ImmutableArray<CodeAc
         protected static ImmutableArray<CodeAction> GetNestedActions(ImmutableArray<CodeAction> codeActions)
             => codeActions.SelectMany(a => a.NestedCodeActions).ToImmutableArray();
 
-        internal (OptionKey, object) SingleOption<T>(Option<T> option, T enabled)
-            => (new OptionKey(option), enabled);
+        internal (OptionKey2, object) SingleOption<T>(Option2<T> option, T enabled)
+            => (new OptionKey2(option), enabled);
+
+        internal (OptionKey2, object) SingleOption<T>(Option<T> option, T enabled)
+            => (new OptionKey2(option), enabled);
```

Comment:
❔ Are the overloads for `Option<T>` and `PerLanguageOption<T>` still necessary?

Reply:
Let me try and remove them and see which tests were using them.

Reply:
Yep, was able to remove all of them. Thanks!